### PR TITLE
Update android TestApp for autolinking

### PR DESCRIPTION
**Pull request by dhei**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

TestApp Android is now on RN 0.60.0 and uses autolinking.

I did smoke testing on analytics/crashes/push/auth modules and they all look good.

### CODE REVIEW

File: TestApp/update-npm-packages.sh [hunk 1]
```diff
@@ -24,5 +24,5 @@ npm install
 echo "Build shared framework..."
 (cd ../AppCenterReactNativeShared/ios && SRCROOT=`pwd` ./build-fat-framework.sh)
 
-echo "Running jetify for react-native 0.60.0-rc3..."
+echo "Running jetify for react-native 0.60.0..."
```
Comment (guperrot) [inline TestApp/update-npm-packages.sh]: We will update RN and forget to update this echo line. Can we remove version or why we do it?
```suggestion
echo "Running jetify"
```

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java [hunk 2]
```diff
@@ -37,19 +26,14 @@ public boolean getUseDeveloperSupport() {
...
-                    new AppCenterReactNativeAuthPackage(getApplication()),
-                    new AppCenterReactNativePushPackage(getApplication()),
-                    new AppCenterReactNativeCrashesPackage(getApplication(), getResources().getString(R.string.appCenterCrashes_whenToSendCrashes)),
-                    new AppCenterReactNativeAnalyticsPackage(getApplication(), getResources().getString(R.string.appCenterAnalytics_whenToEnableAnalytics)),
-                    new AppCenterReactNativePackage(getApplication()),
-                    new RNFSPackage(),
-                    new ImagePickerPackage(),
-                    new AsyncStoragePackage()
-            );
+            @SuppressWarnings("UnnecessaryLocalVariable")
```
Comment (guperrot) [inline TestApp/android/app/src/main/java/com/testapp/MainApplication.java]:
```suggestion

            @SuppressWarnings("UnnecessaryLocalVariable")
```

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java [hunk 3]
```diff
@@ -37,19 +26,14 @@ public boolean getUseDeveloperSupport() {
...
+            List<ReactPackage> packages = new PackageList(this).getPackages();
```
Comment (guperrot) [inline TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: inline variable

Comment (dhei) [reply TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: This is code generated by `react-native init`, so probably okay to keep it.

File: TestApp/android/app/src/main/java/com/testapp/MainApplication.java [hunk 4]
```diff
@@ -37,19 +26,15 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-            return Arrays.asList(
-                    new MainReactPackage(),
-                    new RNGestureHandlerPackage(),
-                    new TestAppNativePackage(getApplication()),
-                    new AppCenterReactNativeAuthPackage(getApplication()),
```
Comment (dhei) [inline TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: Currently we have a limitation that `AppCenterReactNativeAuthPackage` must be started before `AppCenterReactNativeDataPackage` so the order matters. 

We will need to figure out if there is a way to keep the order with autolinking.

Comment (jaelim-ms) [reply TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: Why do we have this limitation?

Comment (guperrot) [reply TestApp/android/app/src/main/java/com/testapp/MainApplication.java]: We don't know when one will remove auth module. Core channel blocks on refreshing token from auth, so if SDK started without auth, channel will sign out assuming auth module was removed.
React-native splits start modules also.

**Pull request merged**